designs/wahid/src/back.js

```diff
@@ -107,6 +107,11 @@ export default (part) => {
   paths.dart.render = false
 
   if (complete) {
+    //Grainline
```

Comment:
We need a combined cutonfold/grainline indicator here, as such:

```js
macro('cutonfold', {
  from: points.cbNeck,
  to: points.cbHem,
  grainline: true,
})
```

Reply:
If we add a cutonfold, won't that give us one back piece instead of two? That is not in accordance with the sewing instructions for Wahid, so in such case we must update the instructions as well :)

Reply:
Again, I believe that the original code and instructions were correct. There should not be a foldline on the Wahid back part. So, I believe that your code change should be reverted (but, I'm hoping that Joost will clarify the situation).

Reply:
Apologies for the tardiness of my response, but @BenJamesBen is correct, the back is not cut on the fold, but joined with a center back seam.

This matters because there's an option to shape the back which does so on the center back seam, so it can't be cut on the fold: https://freesewing.org/docs/patterns/wahid/options/centerbackdart

So yeah, no cut on fold, only grainline.